src/test/java/cmu/pasta/mu2/fuzz/FileMutantFilterTest.java

```diff
@@ -0,0 +1,63 @@
+package cmu.pasta.mu2.fuzz;
+
+import cmu.pasta.mu2.MutationInstance;
+import cmu.pasta.mu2.instrument.Mutator;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FileMutantFilterTest {
+
+    static final String fileName = "testFileForFilter.txt";
+    static File testFile;
+    static FileWriter writer;
+
+    static final List<MutationInstance> muts = new ArrayList<>(Arrays.asList(
+            new MutationInstance("ExampleClass", Mutator.I_ADD_TO_SUB, 1, 1, "ExampleClass.java"),
+            new MutationInstance("ExampleClass", Mutator.ARETURN_TO_NULL, 2, 2, "ExampleClass.java"),
+            new MutationInstance("ExampleClass2", Mutator.I_DIV_TO_MUL, 3, 2, "ExampleClass2.java"),
+            new MutationInstance("ExampleClass2", Mutator.D_SUB_TO_ADD, 4, 42, "ExampleClass2.java")
+    ));
+
+    @Before
+    public void createFile() throws IOException {
+        testFile = new File(fileName);
```

Comment:
Can you replace this with `File.createTempFile()` instead, so that the file created for tests is unique and doesn't persist in the system for too long.